### ISSUE TITLE
Pipe mismatch with tutorials

### ISSUE THREAD

**Issue by mamcisaac** [OpenIntroStat/oilabs-tidy#115]
The tutorials have been updated to use the native pipe, but the labs still use %>%

This will confuse students that are using both the tutorials and labs.

**Comment by mine-cetinkaya-rundel** [comment]
Agreed, we can update these. I wonder about this update right now (middle of semester for many) vs. at the end. Do you have any suggestions?

**Comment by mamcisaac** [comment]
I'm happy either way. I think the mismatch is already a little confusing, so I don't think a change to the native pipe in the labs would be too problematic at this point.

**Comment by nicholasjhorton** [comment]
Any chance that the switch could take place now? I'd be glad to assist with the process.

**Comment by mine-cetinkaya-rundel** [comment]
Thanks @nicholasjhorton!
